### PR TITLE
Change learn more label for programming model step

### LINKED ISSUE #3547: Node v4 model: Clarify what "Learn more" is about
image

**Request:** Change to "Learn more about Model V4"

### PR DESCRIPTION
**Pull request by nturinski**
Fixes #3547 

image

### CODE REVIEW

File: src/commands/createNewProject/ProgrammingModelStep.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { AzureWizardPromptStep, IAzureQuickPickItem, IAzureQuickPickOptions, nonNullValue, openUrl } from '@microsoft/vscode-azext-utils';
-import { learnMoreQp, nodeProgrammingModelSetting } from '../../constants';
```
Comment (bwateratmsft) [inline src/commands/createNewProject/ProgrammingModelStep.ts:7]: Is the `learnMoreQp` constant still referenced by anything else or can it be removed?

File: src/commands/createNewProject/ProgrammingModelStep.ts [hunk 2]
```diff
@@ -29,7 +29,9 @@ export class ProgrammingModelStep extends AzureWizardPromptStep<IProjectWizardCo
             }
         });
 
+        const learnMoreQp = { label: localize('learnMore', '$(link-external) Learn more about Model V4...'), description: '', data: undefined };
```
Comment (bwateratmsft) [inline src/commands/createNewProject/ProgrammingModelStep.ts:32]: Does the `IAzureQuickPick` lack a way to override the "Learn More" text? That would be a good addition to the API that could simplify this code.

File: src/commands/createNewProject/ProgrammingModelStep.ts [hunk 3]
```diff
@@ -29,7 +29,9 @@ export class ProgrammingModelStep extends AzureWizardPromptStep<IProjectWizardCo
...
         if (this._learnMoreLink) {
+
```
Comment (bwateratmsft) [inline src/commands/createNewProject/ProgrammingModelStep.ts:34]: Nit: Spurious newline

**Review by bwateratmsft — APPROVED** [review]

File: src/commands/createNewProject/ProgrammingModelStep.ts (hunk 1 above)
Comment (nturinski) [reply src/commands/createNewProject/ProgrammingModelStep.ts:7]: Still used, but only in one other place. I'll just move it to there.

File: src/commands/createNewProject/ProgrammingModelStep.ts (hunk 2 above)
Comment (nturinski) [reply src/commands/createNewProject/ProgrammingModelStep.ts:32]: I think at the moment we only have a Learn More `MessageItem` for notification dialogs and a Learn More `QuickInputButton` which is that little ? button on the top right of the quickpick window: 
image

Today, we don't have a standardized way of adding `learn more...` to quick picks, but I think it makes sense to add that to the API along with a way to override the learn more text.

**Review by bwateratmsft — APPROVED** [review]

**Pull request merged**